boards/firefly/README.md

```diff
@@ -0,0 +1,111 @@
+Zolertia Firefly platform
```

Comment:
Very nice documentation but IMHO it would be better to put it either at the beginning of board.h or in a separate `doc.txt` file.